mission-and-vision.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Eiffel</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="./images/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./images/favicon/favicon-16x16.png">
+    <link rel="manifest" href="/manifest.json">
+    <link rel="mask-icon" href="./images/favicon/safari-pinned-tab.svg" color="#5bbad5">
+    <link rel='stylesheet' href='./css/index.css'>
+    <meta name="theme-color" content="#ffffff">
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <a class="header-logo no-underline-hover" href="/">
+          <img src="images/eiffel-logo.png" class="header-logo-image"/>
+        </a>
+
+        <nav class="header-nav">
+          <a class="header-footer-nav-link" href="https://github.com/eiffel-community/eiffel">Protocol</a>
+          <a class="header-footer-nav-link" href="./community.html">Community</a>
+          <a class="header-footer-nav-link" href="./mission-and-vision.html">Mission and Vision</a>
+          <a class="header-footer-nav-link" href="./summit.html">Eiffel Summit</a>
+          <a class="header-footer-nav-link no-underline-hover" href="https://www.youtube.com/channel/UC3Zx84ecQ1jyGSc9709xrZg">
+            <img src="images/youtube-channel-icon.png" class="header-nav-link-image"/>
+          </a>
+          <a class="header-footer-nav-link no-underline-hover" href="https://github.com/eiffel-community">
+            <img src="images/github-organization-icon.png" class="header-nav-link-image"/>
+          </a>
+          <a class="header-footer-nav-link no-underline-hover" href="https://groups.google.com/forum/#!forum/eiffel-community">
+            <img src="images/google-group-icon.png" class="header-nav-link-image"/>
+          </a>
+        </nav>
+      </div>
+    </header>
+    <section>
+      <div class="container bg-shade">
+        <h1 class="section-heading">Mission</h1>
+        <p class="centered-text">The Eiffel community's mission is to provide seamless transparency and interconnectedness throughout polyglot and multi-organizational continuous delivery pipelines.</p>
+      </div>
+    </section>
+
+
+    <section>
+      <div class="container">
+        <h1 class="section-heading">Vision</h1>
+        <p class="centered-text">The vision for achieving our mission is three-fold:</p>
+        <ul>
+          <li>A shared vocabulary and protocol for technology agnostic messaging between all actors in the pipeline.</li>
+          <li>Data storage and analysis services providing perfect traceability of what was done when, how, where, why and by whom for every software version, from commit to finished release candidate.</li>
```

Comment:
I know it is a vision but using wording such as perfect can in my experience sometimes give unrealistic expectations and in some contexts it comes across as naive. 

My Suggestions: meticulous, 'a very high degree of'. 

Some say "Perfect is the enemy of good", but I guess its doesn't apply as well to less tangible stuff like visions and such, as it does to code and design. :)

Reply:
That's a fair point, I can buy that.